public_html/assets/js/aws-s3-explorer.js

```diff
@@ -217,9 +219,8 @@ $(function () {
     //update view when url-hash changes
     $( window ).on( 'hashchange', function( e ) {
         if (!prefix.endsWith("/")) {
-            prefix = window.location.hash.substr(1);
-            prefix = prefix.substr(0, prefix.lastIndexOf("/") + 1);
-
+            prefix = window.location.hash.substrring(1);
```

Comment:
Too many r's me hearty 🏴‍☠️

```suggestion
            prefix = window.location.hash.substring(1);
```